examples/all_in_one/slo-exporter/conf/slo_exporter.yaml

```
@@ -0,0 +1,48 @@
+webServerListenAddress: "0.0.0.0:8001"
+maximumGracefulShutdownDuration: "10s"
+afterPipelineShutdownDelay: "1s"
+
+pipeline: ["tailer", "relabel", "eventKeyGenerator", "dynamicClassifier", "sloEventProducer", "prometheusExporter"]
+
+modules:
+
+  tailer:
+    tailedFile: "/logs/access_log"
+    follow: true
+    reopen: true
+    positionFile: ""
+    positionPersistenceInterval: "2s"
+    loglineParseRegexp: '^(?P<ip>[A-Fa-f0-9.:]{4,50}) \S+ \S+ \[(?P<time>.*?)\] request-method="(?P<httpMethod>[^"]+)"\s+request-uri="(?P<requestUri>[^"]+)"\s+request_query=".*" (?P<statusCode>\d+) \d+ "(?P<referer>.*?)" uag="(?P<userAgent>[^"]+)" "[^"]+" ua="[^"]+" rt="(?P<requestDuration>\d+(\.\d+)??)"'
+    emptyGroupRE: '^-$'
+
+  relabel:
+    eventRelabelConfigs:
+      # Drop events with unwanted status codes
+      - source_labels: ["statusCode"]
+        regex: "30[12]|40[045]|411"
```

```suggestion
        regex: "30[12]|40[045]|411|408|499"
```

408 & 499 are special cases which almost always needs to be ingnored.